<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicData</class>
 <widget class="QDialog" name="BasicData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="UM7">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM19">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl24">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>750</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl22">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>690</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM21">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>660</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM13">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl10">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM23">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>720</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl16">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl18">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>690</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl21">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>660</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl17">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>540</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Quit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>850</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl20">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM14">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>450</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM15">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>480</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM20">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM12">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl23">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl25">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>780</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl15">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl19">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM16">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM17">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM18">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>570</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM8">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl9">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM24">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>750</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM25">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>780</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UM1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val15">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>480</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val16">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val17_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val17">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>540</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val18">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val19">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>600</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val20">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>630</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val21">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>660</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val22">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>690</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val23">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>720</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val24">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>750</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val25">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>780</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Val0">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Environmental">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>810</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Environmental</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Basic">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>810</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Basic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Orbital">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>810</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Orbital</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Quit</sender>
   <signal>clicked()</signal>
   <receiver>BasicData</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>751</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Basic</sender>
   <signal>clicked()</signal>
   <receiver>BasicData</receiver>
   <slot>Basic()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>819</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Environmental</sender>
   <signal>clicked()</signal>
   <receiver>BasicData</receiver>
   <slot>Environmental()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>819</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Orbital</sender>
   <signal>clicked()</signal>
   <receiver>BasicData</receiver>
   <slot>Orbital()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>819</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>Basic()</slot>
  <slot>Orbital()</slot>
  <slot>Environmental()</slot>
 </slots>
</ui>
